### PR TITLE
fix: Updated the logo across the entire application

### LINKED ISSUE #251: Update logo across our apps
### Feature Description

Using the new logo on Figma, update the files we have in the static folder for both `apps/dashboard` and `apps/classroomio-com`.

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [ ] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by Chifez**
## What does this PR do?
This PR updates the entire logo across the application

Fixes #251

Screenshot (996)
Screenshot (997)
Screenshot (998)
Screenshot (999)
Screenshot (1000)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- run on localhost

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

File: apps/classroomio-com/src/app.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
     <!-- <link rel="stylesheet" href="global.css"> -->
     <link rel="manifest" href="/manifest.json" crossorigin="use-credentials" />
     <link rel="icon" type="image/png" href="/favicon.ico" />
-    <link rel="icon" type="image/png" sizes="32x32" href="/logo-32.png" />
+    <link rel="icon" type="image/svg" sizes="32x32" href="/new-logo.svg" />
```
Comment (rotimi-best) [inline apps/classroomio-com/src/app.html]: @Chifez Please maintain the same name. You just need to update the content but keep the file name.

File: apps/classroomio-com/static/manifest.json [hunk 2]
```diff
@@ -1,25 +1,15 @@
...
-			"src": "logo-192.png",
-			"sizes": "192x192",
-			"type": "image/png"
-		},
-		{
-			"src": "logo-512.png",
-			"sizes": "512x512",
-			"type": "image/png"
-		}
-	]
```
Comment (rotimi-best) [inline apps/classroomio-com/static/manifest.json:24]: @Chifez You shouldn't remove these icons.

You need to make that you download the right resolution from the figma and add that into the repo.

Please make sure to keep also the file name so we can use it when we need to select large images or smaller images. Also look into the static folder for all `logo-*` those are the images you should update not the code.

In addition please we need to have different types of images like the one with white background, blue background and transparent background. We will use it for different purposes.

File: apps/docs/theme.config.tsx [hunk 3]
```diff
@@ -90,33 +90,33 @@ const config: DocsThemeConfig = {
...
+          //   height: 32,
+          //   alt: 'ClassroomIO Docs Og Image Alt',
+          //   type: 'image/png'
+          // },
+          // {
+          //   url: `${process.env.imagePath}/img/new-logo.svg`,
+          //   width: 192,
+          //   height: 192,
+          //   alt: 'ClassroomIO Docs Og Image Alt',
+          //   type: 'image/png'
```
Comment (rotimi-best) [inline apps/docs/theme.config.tsx]: please uncomment this. You can't comment  these different resolutions of images, they are used by the browser.

**Review by rotimi-best — CHANGES_REQUESTED** [review]

File: apps/docs/theme.config.tsx (hunk 3 above)
Comment (Chifez) [reply apps/docs/theme.config.tsx]: yea i am currently reverting all commits to follow up on it based on your reviews

**Pull request closed without merging**